cli/nep_test/nep17_test.go

```diff
@@ -64,14 +65,26 @@ func TestNEP17Balance(t *testing.T) {
 		e.CheckNextLine(t, "^\\s*$")
 	})
 	t.Run("all accounts", func(t *testing.T) {
+		getUtilityTokenBalance := func(acc util.Uint160) (*big.Int, uint32) {
+			gasHash, err := e.Chain.GetNativeContractScriptHash(nativenames.Gas)
+			require.NoError(t, err)
+
+			gasState := e.Chain.GetContractState(gasHash)
+			require.NotNil(t, gasState)
+
+			lub, err := e.Chain.GetTokenLastUpdated(acc)
```

Comment:
I don't see how call to last updated height is connected with this race. Could you explain?